node.sh

```diff
@@ -0,0 +1,53 @@
+package: node
+version: "v9.8.0"
+build_requires:
+  - curl
+prefer_system: "(?!slc5)"
+prefer_system_check: |
+  which node || { echo "node is missing"; exit 1; }
+  if [ `$(brew --prefix node)/bin/node --version | tr -d v | awk -F \. {'print substr(0 $1, length($1), length($1) + 1) substr(0 $2, length($2), length($2) + 1)'}` -le 0809 ]; then echo "higher node version required"; exit 1; fi
```

Comment:
How about using the `verge` macro we have in swig.sh for doing the same thing?

Reply:
Sure, is there any documentation how `verge` works?

Reply:
I would say it's rather obvious:

```
  verge() {
      [  "$1" = "`echo -e "$1\n$2" | sort -V | head -n1`" ]
  }
```

I.e. it checks if the first entry is the same as the one you would get with a natural sort (`sort -V`).

Reply:
I used what Dario proposed in previous comment